pyproject.toml

```diff
@@ -90,6 +92,9 @@ dependencies = [
   "spectres",
   "tqdm",
   "requests",
+  "dense_basis@git+https://github.com/kartheikiyer/dense_basis#egg=9c404ff",
+  "statsmodels",
+  "scikit-learn",
```

Comment:
Weren't some of these getting removed? `scikit-learn` is a huge dependency to hitch to.